compiler/damlc/daml-visual/src/DA/Daml/Visual.hs

```diff
@@ -138,7 +138,7 @@ startFromUpdate seen world update = case update of
     LF.UCreate tpl _ -> Set.singleton (ACreate tpl)
     LF.UCreateInterface{} ->
       error "Interfaces are not supported"
-    LF.UExercise tpl choice _ _ -> Set.singleton (AExercise tpl choice)
+    LF.UExercise tpl choice _ _ _dynamic -> Set.singleton (AExercise tpl choice)
```

Comment:
Not super familiar with this part of the codebase, but seems like this could be useful information to visualise - perhaps a todo/ticket if you agree?

Reply:
I've added another TODO for this. I don't think we've updated this in a while, though, and there are still TODOs from interfaces and no one has complained, so we might end up dropping it anyway.